src/pudl/package_data/eia923/column_maps/energy_storage.csv

```diff
@@ -0,0 +1,71 @@
+﻿year_index,2014,2015,2016,2017,2018,2019,2020,2021,2022,2023
+plant_id_eia,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id
+is_combined_heat_power,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant,combined_heat_and_power_plant
```

Comment:
I mention this in #3512, but this column shows up in other tables as just `combined_heat_power`. Based on how we've treated booleans in the previous table updates I think it makes sense for this to have the `is_` prefix. 

If you agree, I'll update it in the other table as well.

Reply:
yes i think that makes sense! although iirc there are several hard-coded column names in the transform step  and in the harvestable entity columns so if you change it be sure to make sure you go track down all of the threads. (also there are many boolean columns that don't conform w/ the `is_` prefix in other EIA tables... i'd love to change em all but that feels very oos for this)

Reply:
Yeah, I figure for now we should just fix 'em as they come up

Reply:
It looks like there is a field called `associated_combined_heat_power` in the EIA860 generator tables. This field feels very similar, the only difference being that the EIA860 tables are reported at the `generator_id` level and the EIA923 tables are reported at the `plant_id_eia` level. This makes sense because generators are *associated* with combined heat and power power plants whereas plants *are* combined heat and power. I'm wondering if we should give these columns the same name.

Reply:
Huh, it also seems like the `combined_heat_power` field doesn't get passed from the raw table to any of the transformed `generation_fuel` tables in PUDL. Not sure what to make of that.

Reply:
it looks like it was renamed to `associated_combined_heat_power` at some point and the column from generation_fuel never got renamed. I think that's a fine name all in all and will be minimally invasive to name this new column `associated_combined_heat_power` and fix the gen fuel name